Dockerfile

```diff
@@ -138,12 +137,15 @@ ENV MOLECULE_EXTRAS="azure,docker,docs,ec2,gce,hetznercloud,linode,lxc,openstack
 
 RUN \
     apk add --update --no-cache --repository http://dl-3.alpinelinux.org/alpine/edge/testing/ ${BUILD_DEPS} ${PACKAGES} \
-    gem install ${GEM_PACKAGES} \
+    gem install --no-ri --no-doc ${GEM_PACKAGES} \
     apk del --no-cache ${BUILD_DEPS} \
     rm -rf /root/.cache
 COPY --from=molecule-builder \
     /usr/src/molecule/dist \
     /usr/src/molecule/dist
 RUN pip3 install ${PIP_INSTALL_ARGS} "molecule[${MOLECULE_EXTRAS}]"
 
+# Test rubocop binary executes
+RUN rubocop --version
```

Comment:
Let's test this in the Tox test env and not in the standard Dockerfile?

Reply:
I like testing in dockerfile but it needs to avoid creating a new docker layer by reusing one of the already existing RUN commands.

Please add it after the rm from line 143 as `&& rubocop --version`.

Reply:
Ok I've done this and added a tests env var so we can expand upon this without the run command becoming unwieldy.